volatility3/framework/constants/__init__.py

```diff
@@ -14,7 +14,7 @@
 from typing import Callable, Optional
 
 import volatility3.framework.constants.linux
-import volatility3.framework.constants.windows
+import volatility3.framework.constants.windows  # noqa: F401
```

Comment:
Hmmm, I'd really like that `import foo as foo` trick to work here instead of the noqa line.  Do we think it will?

Reply:
`import as ...` might even break it? (never tried it but when importing it `as something` i'd expect the constants to then be on `something` e.g. `something.CONSTANT` and not in the scope of the file)

Reply:
AFAICT and imo the cleanest way would probably be adding a `__all__` to those 2 files and then doing an `import * from ...`

Reply:
Yeah, I've worked hard to keep `import *` out of the codebase, I'm not going to give people an example.  I assume then that changing this to `from volatility3.framework.constants import windows as windows` to the end, didn't resolve the issue?  The page you'd link earlier seemed to imply it would?

Reply:
i never actually tried it

Reply:
`from volatility3.framework.constants import windows as windows` seems to work, applied here:

https://github.com/volatilityfoundation/volatility3/pull/1364/commits/f2771bce87e80c7a988247f855e5afa117753654